app/views/users/_following_modal.html.erb

```diff
@@ -1,9 +1,8 @@
-<div class="modal-div">
-
-  <!-- Text to trigger modal -->
-  <i class="glyphicon glyphicon-hand-right"></i> <a data-toggle="modal" href="#following">Following:</a> <span class="badge"><%= following.count %></span>
+<!-- Text to trigger modal -->
+<span class="badge"><%= following.count %></span>
+<i class="glyphicon glyphicon-hand-right"></i> <a data-toggle="modal" href="#following">Following</a>
```

Comment:
Bootstrap 3 uses the `<span>` element instead of the `<i>` element. Small, picky thing.